R/param-estimates.R

```diff
@@ -5,24 +5,48 @@
 
 #' Parses parameter estimates table
 #'
-#' Returns a tibble containing parameter estimates from a model.
-#' Currently can only take a `bbi_{.model_type}_summary` object, as output from `model_summary()`.
+#' Returns a tibble containing parameter estimates from a model or batch of models.
+#' Takes either a `bbi_{.model_type}_summary`, `bbi_summary_list`, or `bbi_summary_log_df` object.
+#' Details about the tibble tibble that is returned are in the return value section below.
+#'
+#' @return
+#' **Single model:** For `bbi_{.model_type}_summary` dispatch returns a tibble with the following columns:
+#'
+#' * parameter_names -- Parameter name ("THETA1", "THETA2", etc.)
+#' * estimate -- Parameter estimate
+#' * stderr -- Standard Error
+#' * random_effect_sd
+#' * random_effect_sdse
```

Comment:
Oh yea I missed those. That's a good point. It's buried in the NONMEM docs, but it's not easy just to link to (because those docs aren't in any central public place that I know of). I'll add the description though.